### DISCUSSION TITLE
Manually adjusting toolbar size

### DISCUSSION THREAD

**Discussion by GithubUser699** [xournalpp/xournalpp#4696] [Q&A]
Hello.
### My Problem / Question
I am wondering if it's possible to manually adjust the toolbar size on linux. I used the Mojave-Light theme for a while which looks like this:

_Pict. 1_
![xournalpp_theme1](https://user-images.githubusercontent.com/74927253/223019401-ec721d90-0646-490b-a04a-5b156de36808.png)

Now I switched to Colloid-Light theme, which looks bigger (and more similar to Adwaita):

_Pict. 2_
![xournalpp_theme2](https://user-images.githubusercontent.com/74927253/223019537-17130a35-2980-4f97-8bd4-851e57c43c09.png)

I'd like the toolbar below that among others contains page information to be much much smaller than it is in _Pict. 2_, similar to the size in _Pict. 1_. Is it somehow possible to achieve that?

### Workarounds I considered
Now, I am aware that I could:

1.  Execute xournalpp with set GTK theme by a) setting an alias for the terminal and b) editing the .desktop file for desktop usage: `GTK_THEME=Mojave-light xournalpp`
2. Edit the theme css itself

Currently I am doing the first, which is alright but not exactly ideal (it is rather strange to apply a completly different theme just for selected applications). Second option would change the appearance of every app. I don't want that though.

### Might be related to…
#2807 might be related. It had been asked early 2021 though and specifically targeted windows – maybe such a feature had been implemented already or there is a more elegant workaround for linux than those I stated?

### System info
OS: Debian GNU/Linux
OS version: 11 (bullseye)
GNOME version: 3.38.5
window manager: X11

### App info
xournalpp version: 1.1.3+dev (nogitfound)
build on: Dec 23 2022, 00:05:59
libgtk version: 3.24.24

### Others
Also, Mojave-light didn't display separators correctly (they were there but transparent). If it's possible to insert additional transparent separators for additional styling purposes, that would be great to know.

**Comment by marr2023** [comment]
it's my problem too, even more so that I use a 15 inch monitor.
